esp_encrypted_img/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.5.0"
+version: "2.5.1"
```

Comment:
Likewise, please update https://github.com/espressif/idf-extra-components/blob/55c79124f1973c565cae1683a4274ed87dc8ac6a/esp_encrypted_img/CHANGELOG.md

(sorry that I forgot to mention this earlier)